src/Symfony/Component/VarDumper/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 -----
 
  * added `RdKafka` support
+ * added `VAR_DUMPER_FORMAT=server` support
+ * prevent replacing the handler when the `VAR_DUMPER_FORMAT` env var is set
```

Comment:
Btw what about considering this behavior as a bug fix? Getting something else than the expected format when using  `VAR_DUMPER_FORMAT` looks unexpected to me.

Reply:
What do you think about this? Bug fix or not bug fix?